discovery-infra/start_discovery.py

```diff
@@ -99,21 +119,28 @@ def fill_tfvars(
 
     machine_cidr_addresses = []
     provisioning_cidr_addresses = []
+    assisted_service_ips = []
```

Comment:
why it is needed? I don't understand this change. Why changing something for ipv6 must require changes for ipv4?